modal/sandbox.py

```diff
@@ -91,7 +91,49 @@ async def _get_logs():
         return data
 
 
+MAX_BUFFER_SIZE = 4096
```

Comment:
I think I was wrong the first time, this limit is too low. Let's do 128 KiB

Reply:
ah makes sense, we can't use the same constant as `asyncio`'s `StreamWriter` because they immediately write to the underlying file and only buffers if there is a `BlockingIOError`.